languages/javascript/config.json

```
@@ -27,6 +27,12 @@
         "concepts": ["booleans-basic"],
         "prerequisites": []
       },
+      {
+        "slug": "arrays",
+        "uuid": "25cb0707-44f8-4800-b993-3fcb2b6d9f61",
+        "concepts": ["booleans-basic"],
```

This field should hold the concepts the exercise aims to have the student learn about:

```suggestion
        "concepts": ["arrays-basic"],
```